python/nano/src/bigdl/nano/deps/openvino/pytorch/pytorch_openvino_utils.py

```diff
@@ -28,7 +28,7 @@ def export(model, input_sample=None, xml_path="model.xml"):
     :param input_sample: torch.Tensor or a list for the model tracing.
     :param xml_path: The path to save openvino model file.
     '''
-    export_to_onnx(model, input_sample, 'tmp.onnx', dynamic_axes=False)
+    export_to_onnx(model, input_sample, 'tmp.onnx', dynamic_axes=True)
```

Comment:
Is there any reason or implications for changing the `dynamic_axes` value? Maybe leaving some comments?

Reply:
Without using ModelOptimizer to convert onnx to openvino as before, `dynamic_axes=True` is not acceptable. Now I changed the way to convert onnx to openvino by ModelOptimizer, it's more powerful and convenient for inference with different batches or resolutions.

Reply:
ok